pynecone/components/overlay/popover.py

```diff
@@ -108,6 +108,9 @@ def create(
         Returns:
             The popover component.
         """
+        if trigger == "click" or trigger == "hover":
```

Comment:
I think the expected type of `trigger` is `pc.Component` because we use it to create the component below. We need to add type annotations for this method to make this more clear. I'm thinking we can change the name of the argument `trigger` to `trigger_comp` or something like that here, so it doesn't get override in the `props`.

Reply:
agree

Reply:
@jiangmiemie Do you want to update this PR to rename the argument? We can merge it in after